powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehavior.java

```
@@ -781,10 +787,15 @@ public OperationListResponse findAllOperationsForUser(final OperationListForUser
             final List<String> activationFlags = fetchActivationFlags(activationId);
 
             final OperationListResponse result = new OperationListResponse();
-            try (final Stream<OperationEntity> operationsForUser = operationRepository.findAllOperationsForUser(userId, applicationIds, activationId, activationFlags.isEmpty() ? null : activationFlags, operationListRequest.pageable())) {
+            try (final Stream<OperationEntity> operationsForUser = operationQueryService.findAllOperationsForUser(userId, applicationIds, activationId, activationFlags.isEmpty() ? null : activationFlags, operationListRequest.pageable())) {
                 operationsForUser.forEach(op -> {
-                    final OperationEntity operationEntity = expireOperation(op, currentTimestamp);
-                    result.add(convertFromEntity(operationEntity));
+                    final OperationEntity operationEntity;
+                    try {
+                        operationEntity = expireOperation(op, currentTimestamp);
+                        result.add(convertFromEntity(operationEntity));
+                    } catch (GenericServiceException e) {
+                        logger.error("Operation expiration failed, operation ID: {}", op.getId());
```

A swallowed exception, let's log it at least for the DEBUG level.